archlinuxcn/rustnet/rustnet.install

```diff
@@ -0,0 +1,9 @@
+post_install() {
+  printf "To run \"rustnet\" without sudo, you may grant Linux capabilities to the binary.\n"
+  printf "You can run following command as root:\n"
+  printf "  setcap 'cap_net_raw,cap_net_admin,cap_sys_admin+eip' /usr/bin/rustnet\n"
```

Comment:
This will be lost on update. Better to provide a `.hook` file the user can `ln -s` to `/etc/pacman.d/hooks`.